README.md

```diff
@@ -1,8 +1,8 @@
 # tqec
 
-Design automation software tools for Topological Quantum Error Correction
+Design automation software tools (frontend and backend) for Topological Quantum Error Correction.
```

Comment:
Frontend and backend delineations are not needed here. This sentence just says what `TQEC` is. Let's add a section in the `README` called `Architecture` where we give a gist of the frontend and backend.

Reply:
I agree. I can add some basic details, but I think a deeper dive is warranted for a separate wiki page.